package.json

```diff
@@ -44,6 +45,13 @@
     "react-spring": "npm:@react-spring/web@9.0.0-rc.3",
     "react-use": "^15.3.4",
     "react-use-measure": "^2.0.2",
+    "rehype-format": "^3.1.0",
+    "rehype-parse": "^7.0.1",
+    "rehype-raw": "^5.0.0",
```

Comment:
some of these can be removed (I had previously added them for testing out different things). For example, `remark-code-blocks` and `rehype-raw` :)

Reply:
Good catch! I'll layer that into the deserialization PR